### DISCUSSION TITLE
Welcome to factorio-mod-downloader Discussions!

### DISCUSSION THREAD

**Discussion by vaibhavvikas** [vaibhavvikas/factorio-mod-downloader#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Artikal-afk** [comment]
Hello, the other day (again) https://re146.dev/factorio/mods/en stopped working and now I found software that could help me, but here's the problem, when I try to download it gives an error - 'NoneType' object has no attribute 'get text'

Perhaps the problem is that I'm from Russia and our beloved Roskomnadzor blocked something again (Accidentally or on purpose, it doesn't matter)

**Reply by vaibhavvikas** [reply]
@Artikal-afk That seems like a valid issue. What I can do is **add a proper message in case re146.dev is down.** Since the application uses the re146.dev to download, any error or being the site unavailable will result in unexpected errors. 

I checked again, the app is working fine.

**Comment by Major-PITA** [comment]
Fortunately my browser (FF) alerted during download, file "factorio_mod_downloader-0.2.1.exe" contains malware.

https://www.virustotal.com/gui/file/a065b9ddbdb80d9bbd1170dcdd425a270a6d0ad18fe1585c029cccd3f39fea40

https://www.hybrid-analysis.com/sample/a065b9ddbdb80d9bbd1170dcdd425a270a6d0ad18fe1585c029cccd3f39fea40

**Reply by vaibhavvikas** [reply]
@Major-PITA This is a known issue, as the scanners are flagging it as false positive. As you can see only few are flagging it as malicious.

Already Reported here: https://github.com/vaibhavvikas/factorio-mod-downloader/issues/7

https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg

I'm going through the solutions to fix it.
